aci/data_source_aci_infraaccnodepgrp.go

```diff
@@ -14,14 +15,81 @@ func dataSourceAciAccessSwitchPolicyGroup() *schema.Resource {
 		ReadContext:   dataSourceAciAccessSwitchPolicyGroupRead,
 		SchemaVersion: 1,
 		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
```

Comment:
you can have a look at leveraging https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/aci/base_attr_schema.go#L42

Reply:
Helpful, Thanks @akinross